core/dbt/task/run.py

```diff
@@ -53,25 +46,6 @@
 from .printer import get_counts, print_run_end_messages
 
 
-class Timer:
```

Comment:
What is the rationale behind replacing this class's functionality? It seems to introduce more risk than leaving it in place.

Reply:
I'm now populating the start and end times for a given hook node to add to the node metadata and the `timings` object, so this `Timer` class is now duplicative.

The reasons I want to 🔪 it:
1. It indents the code an extra level, which I'm not a giant fan of
2. Duplicative work as outlined above
3. It seems to be only used here, so probably safe to rip out: https://github.com/search?q=org%3Adbt-labs+%22with+Timer%22+language%3APython&type=code&l=Python